src/main.ts

```diff
@@ -109,10 +109,26 @@ async function analyzeCode(
   for (const file of parsedDiff) {
     if (file.to === "/dev/null") continue; // Ignore deleted files
     for (const chunk of file.chunks) {
+      const validLineNumbers = new Set<number>();
+      chunk.changes.forEach((change: parseDiff.Change) => {
+        if ("ln" in change && change.ln) validLineNumbers.add(change.ln);
```

Comment:
Correct, but it does not pass the typescript compiler check unfortunately and gives an error in the editor. It might be configurable though through the tsconfig options.